msibi/tutorials/propane/propane.ipynb

```diff
@@ -57,18 +58,9 @@
    },
    "outputs": [],
    "source": [
-    "# # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #\n",
-    "# # # # # # WARNING : ENTERING DANGEROUS TERRITORY # # # # # # \n",
-    "# # # # # # # # TRESPASSERS WILL BE SHOT ON SIGHT # # # # # # #\n",
-    "# # # # # # # # # SURVIVORS WILL BE SHOT AGAIN # # # # # # # #\n",
-    "# # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #\n",
-    "#\n",
-    "# os.system('rm state*/* rdfs/C3* f_fits.log state*/log.txt')\n",
-    "# os.system('rm state*/err.txt')\n",
-    "# os.system('rm state*/query.dcd')\n",
-    "#\n",
-    "# # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #\n",
-    "# # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #"
+    "os.system('rm state*/* rdfs/C3* f_fits.log state*/log.txt')\n",
+    "os.system('rm state*/err.txt')\n",
+    "os.system('rm state*/query.dcd')"
```

Comment:
You can get rid of the "optional...warning:. Just add a note about what it's doing, e.g., "Remove files generated during CG simulations."